webapp/src/cloud.ts

```diff
@@ -353,7 +353,7 @@ function getLocalCloudHeaders(allHdrs?: Header[]) {
     if (!auth.hasIdentity()) { return []; }
     if (!auth.loggedIn()) { return []; }
     const userId = auth.userProfile()?.id;
-    return (allHdrs || workspace.getHeaders(true))
+    return (allHdrs || workspace.getHeaders(true/*withDeleted*/, false/*filterByEditor*/))
```

Comment:
This code was filtering out non-skillmap projects when the editor was loaded in the skillmap iframe. But the cloud was sending down all headers, not just skillmap activities. This resulted in an insidious issue where the client would think all non-skillmap remote projects were new projects and try to import them. This would fail with couchdb conflicts.